src/test/java/fi/metatavu/kuntaapi/test/server/integration/ptv/ElectronicServiceChannelInTestsIT.java

```diff
@@ -117,9 +114,9 @@ public void updateElectronicServiceChannelChanges() throws IOException, Interrup
       createPhone("en", "Phone", "+358", "12345-FAKE", "Charged", "Testing", false, "Test phone"),
       createPhone("en", "Fax", "+258", "54321-FAKE", "Free", "Testing fax", true, "Test fax")
     ));
-    kuntaApiResource.setUrls(createLocalizedValue("en", "URL", "www.example.com"));
+    kuntaApiResource.setWebPages(createWebPages("en", "WebPage", "www.example.com", "www.example.com", null));
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "www.example.com" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)